dev/bin/localconfig-example

```diff
@@ -4,7 +4,6 @@
 # If you want to use it, make a copy of this named "localconfig" in this same folder.
 
 MANAGED_CONNECTORS_CLUSTER_ID=mc-cluster-id
-MANAGED_KAFKA_INSTANCE_NAME=mk-local
```

Comment:
Is it worth adding a note/sentence that your `localconfig` can have this line removed too?

Reply:
afaik this the localconfig is deleted/rewritten at every execution. No need to specify that I think